Rewrite (most of) the Overview pages in the Docs (#2996)
PostHog/posthog.com: contents/docs/index.md

---
title: PostHog overview
sidebar: Docs
showTitle: true
---

> PostHog is the product analytics suite you can self-host. With our open source platform, customer data never has to leave your infrastructure.

Reviewer: ```suggestion
PostHog is the all-in-one product analytics suite you can self-host. With our open source platform, customer data never has to leave your infrastructure.
```

> We believe the era of third-party product analytics software will eventually come to end, and that there should be an alternative to sending your customers' personal information and usage data to third parties. PostHog gives you full control over all the data from your users, while allowing you to do powerful analytics.

Reviewer: ```suggestion
We believe the era of third-party product analytics software will eventually come to end, and that there should be an alternative to sending your customers' personal information and usage data to third parties. PostHog gives you full control over all your user data, while enabling you to do powerful analytics.
```

Without a self-hosted solution like PostHog, companies who cannot send their data to third-party tracking services for cost or privacy reasons often end up with a complex data pipeline into a data warehouse, with an analytics platform on top. Data analysts end up being the only people who understand it.

> PostHog solves that. We let every person in the company have easy access to product analytics that they can understand and use independently, even at a massive scale, all without sending data to third parties.

Reviewer: ```suggestion
PostHog solves that. We let every person in the company have easy access to product analytics that they can understand and use independently, even at a massive scale, all without sending data to third parties. Additionally, you can run analysis through a straightforward, self-serve interface -- no SQL required!
```

## What can I use PostHog for?

> PostHog providers everything product-led teams need in one place, including:

Reviewer: ```suggestion
PostHog provides everything product-led teams need in one place, including:
```

- Product Analytics
- Session Recording
- Feature Flags
- Heatmaps
- Multivariate A/B tests
- Event pipelines
- Export to data warehouse

Using these products independently across different providers means sending data to multiple third parties that don't talk to each other. With PostHog, you can keep all your product insights on one platform - and we never need to see your users' data. 

## How to use these Docs

### Deploy PostHog

There are 2 ways of using PostHog:

* [PostHog Cloud](/docs/cloud) - get started immediately, we host and manage everything for you
* [Self-host](/docs/self-host) - user data stays on your infrastructure, you host and manage your own instance

### Integrate PostHog

You can capture events and send data to PostHog in 3 different ways:

**Snippet**

Using our [HTML snippet](/docs/integrate/client/snippet-installation) is the quickest way to get started if you're using PostHog to track a website. You can also start trying PostHog with no code using [our bookmarklet](/docs/integrate/client/snippet-installation#get-started-with-no-code).

**Integrations**

> We have PostHog libraries written in all major programming languages, as well as integrations available with services like [Segment](/docs/integrate/third-party/segment), [Slack](/docs/integrate/webhooks/slack), and [Sentry](/docs/integrate/third-party/sentry).

Reviewer: ```suggestion
We have PostHog libraries written in all major programming languages, as well as integrations available with services like [Segment](/docs/integrate/third-party/segment), [Slack](/docs/integrate/webhooks/slack), and [Sentry](/docs/integrate/third-party/sentry). Visit our [Integration library for a full list of integrations](/integrations), or head to [our GitHub repo](https://github.com/PostHog) to start building your own. 
```
